photon-diagnostics/src/main/scala/com/linkedin/photon/ml/Evaluation.scala

```diff
@@ -64,40 +65,42 @@ object Evaluation extends Logging {
 
     // Compute regression facet metrics
     model match {
-      case r: Regression =>
+      case _: Regression =>
         val regressionMetrics = new RegressionMetrics(scoreAndLabel)
-        metrics ++= Map[String, Double](MEAN_ABSOLUTE_ERROR -> regressionMetrics.meanAbsoluteError,
-                                        MEAN_SQUARE_ERROR -> regressionMetrics.meanSquaredError,
-                                        ROOT_MEAN_SQUARE_ERROR -> regressionMetrics.rootMeanSquaredError)
```

Comment:
It is, and it's apparently fixed ... only in 2.12, according what I could gather on the internet, which is disappointing. But again, the point is to try to make it clearer what is error and what is warning when looking at the results of the build.